config/controller/admin/enhancer.config.php

```diff
@@ -36,16 +36,23 @@
                 'required',
             ),
         ),
-        'format' => array(
-            'label' => __('Format:'),
-            'form' => array(
-                'type' => count($formats) === 1 ? 'hidden' : 'select',
-                'options' => $formats,
-                'value' => $current,
-            ),
-        ),
     ),
     'preview' => array(
         'view' => 'noviusos_slideshow::admin/enhancer/preview',
     ),
 );
+
+// Don't display list of format if we only have one format
+if (count($formats) > 1) {
+    $config['fields']['format'] = array(
+        'label' => __('Format:'),
+        'form'  => array(
+            'type'    => count($formats) === 1 ? 'hidden' : 'select',
```

Comment:
It was a mistake i made while merging projects. It should be corrected now and the enhancer preview will hide formats if there is only one defined.